Source/cairo/WaylandCairoSurface.m

```diff
@@ -36,43 +39,45 @@
 #include <fcntl.h>
 #include <sys/mman.h>
 
-#define GSWINDEVICE ((struct window *)gsDevice)
+#define GSWINDEVICE ((struct window *) gsDevice)
 
 /* Linux specific version */
 static int
 os_create_anonymous_file(off_t size)
 {
-    static const char template[] = "/weston-shared-XXXXXX";
-    const char *path;
-    char *name;
-    int fd;
-
-    path = getenv("XDG_RUNTIME_DIR");
-    if (!path) {
-	errno = ENOENT;
-	return -1;
+  static const char template[] = "/gnusteo-shared-XXXXXX";
```

Comment:
Is this a typo or intentional? Most likely it should be "gnustep"

Reply:
:) indeed, I am going to amend the previous commit